### PR TITLE
Added click-to-copy. Adjusted styling.

### PR DESCRIPTION
**Pull request by michaelhollman**
My initial project for skunkworks didn't go super hot. Ended up being a dead-end research project. So I needed to mentally detox and decided to do something easy and satisfying.

This adds click-to-copy using [ZeroClipboard](http://zeroclipboard.org/). It also adjust the styling/layout of the emotes.

I think we technically need to include the MIT license now...?

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -29,6 +30,7 @@ type Emoticon struct {
 
 func main() {
 	http.HandleFunc("/", handle)
+	http.HandleFunc("/zeroclipboard/", fileHandle)
```
Comment (bclymer) [inline main.go]: You should consider setting up a `static` folder, go routing supports this. It just auto serves everything from the folder you give it.

File: main.tmpl [hunk 2]
```diff
@@ -52,7 +64,14 @@
 				}
 			}
 		};
+		ZeroClipboard.config({
+			swfPath: document.location.origin + "/zeroclipboard/ZeroClipboard.swf",
```
Comment (bclymer) [inline main.tmpl]: We're loading a shockwave file now?

Comment (michaelhollman) [inline main.tmpl]: It's the dark magic behind ZeroClipboard.

File: main.go (hunk 1 above)
Comment (michaelhollman) [inline main.go]: like...

``` go
fs := http.FileServer(http.Dir("static"))
http.Handle("/static/", http.StripPrefix("/static/", fs))
```

?

**Comment by tysonstewart** [comment]
Wouldn't hurt to add the MIT license. Can you add that to your PR? 

Also, it looks like the /static directory and all contents will need to be deployed along with the binary. Is that correct?

**Comment by michaelhollman** [comment]
@tysonstewart I believe you're correct about the static directory.

**Comment by tysonstewart** [comment]
I can push it live whenever you think it's ready to go.

**Comment by michaelhollman** [comment]
I'm not planning on doing anything else in this PR, so feel free to push it whenever it's convenient for you.

**Pull request merged**